students/grzegorz_bajorski/lesson_10_organizing_files/selective_copy.py

```diff
@@ -0,0 +1,14 @@
+import shutil
+import os
+
+
+dest_dir = "copy_folder/"
+os.mkdir(dest_dir)
+
+for filename in os.listdir():
+    if filename.endswith('.pdf'):
+        print(filename)
+        shutil.copy(filename, dest_dir)
+    if filename.endswith('.txt'):
```

Comment:
instead of the second if better is used to use elif statement, so in the case when first if will be True second one will be not checked. You can also combine this two if statement into one.